lib/spec/openapi/utils.js

```diff
@@ -326,8 +338,13 @@ function resolveResponse (fastifyResponseJson, produces, ref) {
       statusCode = statusCode.toUpperCase()
     }
 
+    const referenceJsonSchema = findReference(definitions, rawJsonSchema.$ref)
```

Comment:
I think this search should happen only if `rawJsonSchema.description` is falsy, not systematically.

Reply:
Since we don't need anything else, can definitely do so for now.